File: openapi/code_samples/javascript/api_v1_execute_tool/post.js
const response = await scalekit.tools.executeTool({
  toolName: 'gmail_send_email',
  identifier: 'user@example.com',
  params: {
    to: 'team@example.com',
    subject: 'Hello from Scalekit',
    body: 'Tool execution succeeded.',
  },
})

File: openapi/code_samples/python/api_v1_execute_tool/post.py
response = scalekit_client.tools.execute_tool(
    tool_name="gmail_send_email",
    identifier="user@example.com",
    params={
        "to": "team@example.com",
        "subject": "Hello from Scalekit",
        "body": "Tool execution succeeded.",
    },
)
